site/app/templates/calendar/Calendar.twig

```diff
@@ -6,6 +6,8 @@
     const curr_month = {{ curr_month }};
     const curr_day = {{ curr_day }};
     const instructor_courses = {{ instructor_courses|json_encode|raw }};
+    const global_items_by_date = {{global_items_by_date|json_encode|raw }};
+    const is_superuser = {{ is_superuser|json_encode|raw }};
```

Comment:
Why is this json encoded and raw?

Reply:
Fixed with 12cbcd8b445807334be6359623f883723fbe7db6